src/platform/datapath_linux.c

```diff
@@ -182,6 +182,11 @@ typedef struct CXPLAT_SOCKET_CONTEXT {
     //
     CXPLAT_LIST_ENTRY PendingSendContextHead;
 
+    //
+    // Lock around the PendingSendContext list.
+    //
+    CXPLAT_LOCK PendingSendContextLock;
```

Comment:
Rename this to something like `SendLock` now?

Reply:
Now that this is only around pending sends and nothing else, likely best to keep it this way.